include/avif/avif.h

```diff
@@ -484,6 +484,8 @@ typedef struct avifPixelAspectRatioBox
 typedef struct avifCleanApertureBox
 {
     // 'clap' from ISO/IEC 14496-12:2022 12.1.4.3
+    // Note that ISO/IEC 23000-22:2024 7.3.6.7 requires the decoded image to be upsampled to 4:4:4 before
+    // clean aperture is applied if a clean aperture size or offset is odd in a subsampled dimension.
```

Comment:
IMPORTANT: AVIF can subsample an odd width or height. This comment should clarify that we relax this requirement for AVIF, similar to the comment at src/avif.c:885:

```
    // AV1 supports odd dimensions with chroma subsampling in those directions, so only look for x and y.
```